local-tests/setup/tinny-environment.ts

```diff
@@ -215,6 +215,24 @@ export class TinnyEnvironment {
       });
     }
 
+    if (globalThis.wasmExports) {
+      console.warn(
+        'WASM modules already loaded. wil overide. when connect is called'
```

Comment:
Spelling: 

'WASM modules already loaded. wil overide. when connect is called'

To: 

'WASM modules already loaded. Will override when connect is called.'